ts/components/governance/wizard_flow.tsx

```diff
@@ -0,0 +1,287 @@
+    margin-bottom: 8px;
+    border: 1px solid #dddddd;
+    background-color: ${() => colors.white};
+    padding: 10px 20px;
+    display: flex;
+    align-items: center;
+    height: 80px;
+`;
+
+const Heading = styled.div`
+    display: flex;
+    flex: 1;
+    align-items: center;
+`;
+
+const JazzIconContainer = styled.div`
+    height: 40px;
+    width: 40px;
+    margin-right: 20px;
+    border: 1px solid #dddddd;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+
+const Input = styled.input`
+  color: ${colors.textDarkSecondary};
+  font-size: 16px;
+  padding-bottom: 3px;
+  border: none;
+  background: none;
+  text-align: right;
+  margin: 0 5px;
+  outline: none;
+  height: 100%;
+  width: 99%;
+
+  /* Chrome, Safari, Edge, Opera */
+  &::-webkit-outer-spin-button,
+  &::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+
+  /* Firefox */
+  [type=number] {
+    -moz-appearance: textfield;
+  }
+`;
+
+const Title = styled.h3`
+    font-size: 18px;
+    line-height: 1.34;
+    margin-right: 5px;
+
+    @media (min-width: 768px) {
+        font-size: 22px;
+    }
+`;
+
+const ZRXAmount = styled.span`
+    color: ${colors.textDarkSecondary};
+    font-size: 16px;
+    margin-right: 15px;
+    flex: 1;
+    display: flex;
+    align-items: center;
+`;
+
+const Notice = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 80px;
+`;
+
+const Bullet = styled.div`
+  height: 10px;
+  width: 15px;
+  margin-right: 15px;
+  background-color: #000000;
+`;
+
+const ConfirmButton = styled(Button)`
+  margin-top: 40px;
+  color: ${() => colors.white};
+`;
+
+const DelegateButton = styled.button`
+  border: none;
+  background: none;
+  color: ${() => colors.textDarkSecondary};
+  text-align: right;
+  margin-bottom: 60px;
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+export const VotingPowerInput: React.FC<IVotingPowerInputProps> = ({ userZRXBalance, onOpenConnectWalletDialog, address, onNextButtonClick, stakingPools, nextEpochStart }) => {
+  const [shouldOpenPoolsDialog, setOpenPoolsDialog ] = React.useState<boolean>(false);
+  const [selectedPool, setSelectedPool] = React.useState<PoolWithStats>();
+  const [isDelegationFlow, setIsDelegationFlow ] = React.useState<boolean>(false);
+  const [ zrxAmount, setZRXAmount] = React.useState<number>(userZRXBalance);
+
+  React.useEffect(() => {
+    setZRXAmount(userZRXBalance);
+  }, [userZRXBalance]);
+
+  const onInput: React.ChangeEventHandler = (event: React.ChangeEvent) => {
+    setZRXAmount(parseInt((event.target as HTMLInputElement).value, 10));
+  };
+  const onChangePool = (fromPoolId: string, toPoolId: string) => {
+    const selectedPoolObj = stakingPools.find(s => s.poolId === toPoolId);
+    setSelectedPool(selectedPoolObj);
+    setIsDelegationFlow(true);
+    onNextButtonClick(true, selectedPoolObj, zrxAmount);
+  };
+
+  return (
+    <>
+    {
+      !!userZRXBalance ?
```

Comment:
super small nit, but Is this double negation needed since we know userZRXBalance is always number type?

Reply:
Now that I see this I don't think the double negation is required.  But one case that needs to be handled here is when the balance is 0. Even then we want to show the input. Added that.